### DISCUSSION TITLE
Unable to install HomeBrew

### DISCUSSION THREAD

**Discussion by ajtruckle** [Homebrew/discussions#657] [Getting started]
I tried the standard command in my **Terminal** as indicated on your homepage:

`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`

But this is what happens:

```
==> This script will install:
/usr/local/bin/brew
/usr/local/share/doc/homebrew
/usr/local/share/man/man1/brew.1
/usr/local/share/zsh/site-functions/_brew
/usr/local/etc/bash_completion.d/brew
/usr/local/Homebrew

Press RETURN to continue or any other key to abort
==> /usr/bin/sudo /usr/sbin/chown -R andrewtruckle:admin /usr/local/Homebrew
==> Downloading and installing Homebrew...
Updating files: 100% (2474/2474), done.
HEAD is now at 24bbfffc6 Merge pull request #10455 from SeekingMeaning/cask-audit-verified
error: Not a valid ref: refs/remotes/origin/master
fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
fatal: the remote end hung up unexpectedly
fatal: early EOF
fatal: index-pack failed
Error: Fetching /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core failed!
fatal: invalid upstream 'origin/master'
Failed during: /usr/local/bin/brew update --force --quiet
```

Please help me resolve this issue. I am using a **MacBook Pro** with latest **macOS**.

**Comment by mpgovinda** [comment]
I resolved the issue just by adding the following

`git config --global user.email yourgitemail@example.com`
Then remove the cellar and homebrew directories from `/usr/local` and re-try the installation.

**Reply by stoufa** [reply]
I tried it but unfortunately, it doesn't work.
The only thing that 'kind of' worked for me is what was mentioned in this [medium article](https://medium.com/macoclock/a-clean-approach-to-installing-homebrew-d17c797fb045).
However, I noticed that the `brew` command can't install anything! Is it a network issue?

**Reply by gromgit** [reply]
> The only thing that 'kind of' worked for me is what was mentioned in this medium article.

Which is basically the method documented by Homebrew here: https://docs.brew.sh/Installation#untar-anywhere

> However, I noticed that the `brew` command can't install anything! Is it a network issue?

Hard to say, since you haven't shown us any error messages that you see.

Also please post the output of `brew config` and `brew doctor`.

**Reply by ajtruckle** [reply]
In the end I gave up on Brew.

**Reply by stoufa** [reply]
@gromgit I faced [this error](https://stackoverflow.com/questions/51120225/git-push-getting-failed-error-rpc-failed-curl-56-libressl-ssl-read-ssl-error) yesterday, however, I managed how to fix it today, it's as simple as switching to `bash` before running that command on `homebrew`'s homepage 😅

**Reply by carlocab** [reply]
That's a very weird fix, given that the command starts with `/bin/bash`. This means it should be running in `bash` already...

**Reply by stoufa** [reply]
I'm still puzzled myself! Apparently, `/bin/bash` isn't enough.
